pom.xml

```diff
@@ -304,7 +309,7 @@
 		</dependency>
 		<dependency>
 			<groupId>org.slf4j</groupId>
-			<artifactId>slf4j-log4j12</artifactId>
+			<artifactId>slf4j-reload4j</artifactId>
 			<version>1.7.36</version>
```

Comment:
Should this be done in gui also?

Reply:
yes but I was going to wait until this was this PR from buildbot was merged first: https://github.com/informatici/openhospital-gui/pull/1123